tests/magick-image/thumbnail.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*
+  Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+  Licensed under the Apache License, Version 2.0.
+*/
+
+import { MagickGeometry } from '@src/types/magick-geometry';
+import { TestImages } from '@test/test-images';
+
+describe('MagickImage#thumbnail', () => {
```

Comment:
I think we also need an extra test that makes sure that the some of the metadata is not available in the resulting image.

Reply:
I added a test for that! FYI Magick.NET is also missing this test.

Reply:
It seems EXIF attributes (so `.getAttribute`, not `.getProfile`) are not removed, also not by `.strip()`. Is this intended behaviour?

Reply:
I try not to look at the Magick.NET tests and just add here what I think that should be added. Those attributes are not removed because they will not be stored to file. Maybe to make sure this is working correctly we could save the thumbnail to a blob and then read it again but I don't know if we really need to do that.